mpxe/sims/mcp23008.py

```diff
@@ -23,11 +25,11 @@ def update_pin_state(self, proj, pin, state):
 
         mcp23008_mask            = mcp23008_pb2.MxMcp23008Store()
         mcp23008_mask.state[pin] = 1
-    
+
         proj.write_store(mcp23008_msg, mcp23008_mask, stores_pb2.MxStoreType.MCP23008)
 
     # Compares pin state against store
+    # pylint: disable=unused-argument
     def assert_store_value_reading(self, proj, pin, state):
-        for i in range(NUM_MCP_PINS):
-            assert(self.states[pin] == state)
-
+        for _i in range(NUM_MCP_PINS):
+            assert self.states[pin] == state
```

Comment:
I'm not quite sure what this function was intended to do (@mitchellostler ?) but to me it looks safe to remove the `for` line entirely and just leave the assert.

Reply:
Yeah should be good to get rid of, function should be just checking state of pin idk why i put a for loop there lol